source/parser.hera

```diff
@@ -5688,20 +5684,49 @@ Init
       }
     }
 
-    module.reorderBindingRestProperty = function(rest, after) {
-      // Swap delimiters of last property and rest so that an omitted trailing comma doesn't end up in the middle
-      if (after.length) {
-        const
-          [restDelim]       = rest.children.slice(-1),
-          lastAfterProp     = after[after.length - 1],
-          lastAfterChildren = lastAfterProp.children,
-          [lastDelim]       = lastAfterChildren.slice(-1)
+    module.reorderBindingRestProperty = function(props) {
+      const names = props.flatMap(p => p.names)
+
+      let restIndex = -1
+      let restCount = 0
+      props.forEach(({type}, i) => {
+        if (type === "BindingRestProperty") {
+          if (restIndex < 0) restIndex = i
+          restCount++
+        }
+      })
 
-        rest = {...rest, children: [...rest.children.slice(0, -1), lastDelim]}
-        after = [...after.slice(0, -1), {...lastAfterProp, children: [...lastAfterChildren.slice(0, -1), restDelim]}]
+      if (restCount === 1) {
```

Comment:
If you wanted to, you could throw a parse error when `restCount > 1`. Maybe beneficial? But not a big deal.